docs/multicluster/user-guide.md

```diff
@@ -231,7 +231,7 @@ spec:
     namespace: antrea-mcs-ns
 ```
 
-* Create `ClusterClaim` and `ClusterSet` in the leader cluster `test-cluster-north`.
+* Create `ClusterClaim` and `ClusterSet` in the leader cluster `test-cluster-north`. A sample is like below, you can also refer to the template [multicluster_clusterset_template.yaml](../../multicluster/config/samples/multicluster_clusterset_template.yaml).
```

Comment:
the path is wrong (benefitted from the link check)

since you need to make a change, maybe also remove "the template":

you can also refer to [multicluster_clusterset_template.yaml](...).

Reply:
done